weather.py

```diff
@@ -35,8 +34,10 @@ def convert_f_to_c(temp_in_farenheit):
     Returns:
         A float representing a temperature in degrees celcius, rounded to 1dp.
     """
-    pass
+    num = float(temp_in_farenheit)
```

Comment:
Minor comment - it's generally good practice to avoid the use of '[magic numbers](https://en.m.wikipedia.org/wiki/Magic_number_(programming)#:~:text=The%20term%20magic%20number%20or,1%20manuals%20of%20the%201960s)'. This helps with code readability, when there are lots of different constants throughout the codebase. 
Otherwise it's very readable - I liked how you broke this down step-by-step.